### PR TITLE
Add full class name substitution in closures

### PR DESCRIPTION
**Pull request by xepozz**
| Q             | A
| ------------- | ---
| Is bugfix?    | ✔️
| New feature?  | ✔️
| Breaks BC?    | ❌
| Depends on  | #9  

Now dumper will dump closures with usage `asPhpString()` method like the following:
image

### CODE REVIEW

File: src/VarDumper.php [hunk 1]
```diff
@@ -438,4 +466,22 @@ private function cleanShortClosureTokens(array $tokens): array
 
         return $tokens;
     }
+
+    public function getUsesParser(): UseStatementParser
```
Comment (samdark) [inline src/VarDumper.php:470]: Should it be public?

File: tests/UseStatementParserTest.php [hunk 2]
```diff
@@ -0,0 +1,78 @@
...
+                    'ArrayHelper' => '\Yiisoft\Arrays\ArrayHelper',
+                    'ArrayableTrait' => '\Yiisoft\Arrays\ArrayableTrait',
+                ],
+            ],
+        ]);
+    }
+
+    public function saveExamplesToTemporaryFile(array $examples): array
+    {
+        static $handles = [];
```
Comment (samdark) [inline tests/UseStatementParserTest.php:67]: Is it a leftover? Doesn't seem to be used...

Comment (xepozz) [reply tests/UseStatementParserTest.php:61]: No it's needed for tests. `usesProvider` provides temporary file that contains code from provider.

Comment (samdark) [reply tests/UseStatementParserTest.php:61]: Need an inline comment then. I bet every person reading the code will be eager to delete these lines.

**Review by samdark — APPROVED** [review]

**Comment by samdark** [comment]
Thank you!

**Pull request merged**